src/dev/flang/fuir/analysis/dfa/DFA.java

```diff
@@ -1849,6 +1864,101 @@ static void setArrayElementsToAnything(Call cl, int argnum, String intrinsicName
         jobj.setField(cl._dfa, jref, Value.UNKNOWN_JAVA_REF);
         return jobj;
       });
+  }
+  static Value newFuzionJavaCall(Call cl) {
+    var rc = cl._dfa._fuir.clazzResultClazz(cl._cc);
+    if (cl._dfa._fuir.clazzBaseName(rc).startsWith("outcome"))
```

Comment:
I think this comparison was introduced by me in the Interpreter a long time ago. This is ugly and error-prone, it must at least be marked using `// NYI: HACK: should properly check if rc is an outcome` or similar, best would be to implement a proper check, i.e., FUIR should know `outcome` and have methods to check if a class is based on that feature.